### PR TITLE
Added option to exclude files

### PR DESCRIPTION
**Pull request by JarFiles**
#176

### CODE REVIEW

File: src/main/java/net/lingala/zip4j/model/ExcludeFileHandler.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package net.lingala.zip4j.model;
+
+import java.io.File;
+
+public interface ExcludeFileHandler {
+
+    boolean isExcluded(File file);
```
Comment (fatmyxtap) [inline src/main/java/net/lingala/zip4j/model/ExcludeFileHandler.java:7]: What about more than one excluded file or directory?

``` 
- root
     -- firstFolder
     -- secondFolder
     -- thirdFolder
     -- firstFile
     -- secondFile
```
For example, I want to exclude firstFolder, secondFolder and firstFile. 
I think I am not able to do it with current solution, right?

**Comment by JarFiles** [comment]
Yes that would be possible. Use ZipParameters#setExcludeFileHandler() and pass an instance of the ExcludeFileHandler. Now create a list of all files which you want to exclude and check if the file matches the file in the method parameter. I am on phone right now, I can post the code here tomorrow.

**Comment by fatmyxtap** [comment]
Oh, got it, thank you!

**Comment by JarFiles** [comment]
I hope everything works. Let me know if there is something wrong :)

**Comment by srikanth-lingala** [comment]
Sorry, I have been a bit busy lately. I will have a look at this tomorrow.

**Comment by fatmyxtap** [comment]
@srikanth-lingala hi, could you please check it?

**Comment by JarFiles** [comment]
Any update?

**Comment by srikanth-lingala** [comment]
Sorry for the delay, but I am going through some of the busiest days now. I will have a look at it in a day or two.

**Comment by srikanth-lingala** [comment]
PR looks fine. I will merge it. I will add some test before making a release. I have created an issue for it #189

**Pull request merged**